src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9973,5 +9973,85 @@ public async Task RecordStruct()
 record  struct  R(int X);
 ");
         }
+
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task FormatListPattern()
+        {
+            var code = @"class C
+{
+    void M() {
+_ = this is[1,2,>=3];
+}
+}";
+            var expectedCode = @"class C
+{
+    void M()
+    {
+        _ = this is [ 1, 2, >= 3 ];
```

Comment:
`_ = this is [1, 2, >= 3]`

Reply:
i think this will be like parens.  and we'll want to have no spaces around them.   similar to `foo[1, 2, 3]` as well.